app/models/comment.rb

```diff
@@ -1,6 +1,7 @@
 class Comment < ActiveRecord::Base
   belongs_to :review
   belongs_to :user
+  belongs_to :pizzera
```

Comment:
A comment does not belong to a pizzeria, it belongs to a review (which belongs to a pizzeria). To get to the pizzeria you need to go through the review (which you do in the `pizzeria` method below).